src/xcode/ENA/ENA/Source/Workers/Logger.swift

```diff
@@ -16,27 +16,75 @@
 // under the License.
 
 import Foundation
+import SwiftyBeaver
 
 let appLogger = Logger()
 
-func log(message: String, level _: LogLevel = .info, file _: String = #file, line _: UInt = #line, function _: String = #function) {
-	NSLog("%@", message)
-}
-
-func logError(message: String, level _: LogLevel = .error, file _: String = #file, line _: UInt = #line, function _: String = #function) {
-	NSLog("%@", message)
-}
-
 class Logger {
-	func log(message _: String, level _: LogLevel = .info, file _: String, line _: UInt, function _: String) {}
+	
+	private let sb = SwiftyBeaver.self
+	
+	private let fileDest = FileDestination()
+	private let consoleDest = ConsoleDestination()
 
-	func getLoggedData() -> Data? {
-		Data()
+	private var logDirectory: URL? {
+		return try? FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: true)
```

Comment:
I suggest to use `.cachesDirectory` instead of `.documentDirectory`

Reply:
Yes, the log files are better placed in the caches directory. Just pushed a new commit to fix this.